src/components/interactive-builder/modals/question/question-form/common/concept-search/concept-search.component.tsx

```diff
@@ -14,16 +14,17 @@ interface ConceptSearchProps {
   defaultConcept?: string;
   onClearSelectedConcept?: () => void;
   onSelectConcept: (concept: Concept) => void;
+  loadConceptToContext?: boolean;
 }
 
 const ConceptSearch: React.FC<ConceptSearchProps> = ({
   label,
   defaultConcept,
   onClearSelectedConcept,
   onSelectConcept,
+  loadConceptToContext = false,
```

Comment:
Is there a better name here like "retainConceptAfterSearch" or something? I find `loadConceptToContext` opaque.